src/resources/js/components/Presentation.vue

```diff
@@ -93,6 +92,18 @@ export default {
           alert.showMessage('error', 'Oops une erreur c\'est produite lors de la désinscription');
         });
     },
+    getPresentation(id) {
+      alert = this.parentRefs.alert;
```

Comment:
Should be in the before mount hook has it needs to be accessed by all the component code, here it still work but breaks the single responsibility convention `each function do one specitif thing but do it well`.
So here the mean of this function is to get a presentation not to set the component environment, as it is for the beforeMount Hook

Reply:
excepted if you have a good reason to have moved it here

Reply:
Is it just for avoid t write this code twice, no other reason